### PR TITLE
Define fuse_session_loop_mt as a macro on uclibc and MacOS

### PR DESCRIPTION
**Pull request by asafkahlon**
On uclibc and MacOS we don't use versioned symbols. Hence,
there's no definition for fuse_session_loop_mt on those cases
and the linker won't be able to resolve calls to fuse_session_loop_mt()

Signed-off-by: Asaf Kahlon <asafka7@gmail.com>

### CODE REVIEW

File: include/fuse_lowlevel.h [hunk 1]
```diff
@@ -471,7 +471,7 @@ struct fuse_lowlevel_ops {
 	 *  - When writeback caching is disabled, the filesystem is
 	 *    expected to properly handle the O_APPEND flag and ensure
 	 *    that each write is appending to the end of the file.
-	 * 
+	 *
```
Comment (yann-morin-1998) [inline include/fuse_lowlevel.h:474]: Spurious, unrelated space cleanup. Ditto below.

File: lib/fuse_versionscript [hunk 2]
```diff
@@ -146,6 +146,7 @@ FUSE_3.2 {
 	global:
 		fuse_session_loop_mt;
 		fuse_session_loop_mt_31;
+                fuse_session_loop_mt_32;
```
Comment (yann-morin-1998) [inline lib/fuse_versionscript:149]: The lines around use leading TABs, not spaces.

**Comment by asafkahlon** [comment]
@yann-morin-1998 fixed your comments, thanks.
